backend-project/small_eod/generic/serializers.py

```diff
@@ -1,11 +1,20 @@
-from rest_framework.serializers import ModelSerializer
+from rest_framework.serializers import ModelSerializer, PrimaryKeyRelatedField
 
 
 class UserLogModelSerializer(ModelSerializer):
+    created_by = PrimaryKeyRelatedField(read_only=True)
+    modified_by = PrimaryKeyRelatedField(read_only=True)
+
+    def get_current_user(self):
```

Comment:
Moim zdaniem lepiej przyjąć, że będzie to wymagane, gdyż wtedy unikniemy błędów, że gdzieś to przeoczyliśmy. Zobacz na #153 gdzie to obsłużyłem.

Reply:
@ad-m - nie mam pojecia jak te Twoje zmiany domergowac do mojego, zeby wszystko bylo fajnie

Reply:
Proszę, wykonane za Ciebie. Możesz pobrać zmiany z swojego brancha, a nie powinno być już konfliktów.